example/main.go

```diff
@@ -32,23 +32,13 @@ func main() {
 	}
 
 	logger := zerolog.New(os.Stdout).With().Timestamp().Logger()
-
-	server, err := baseapp.NewServer(
-		config.Server,
-		baseapp.DefaultParams(logger, "exampleapp.")...,
-	)
-	if err != nil {
-		panic(err)
-	}
+	zerolog.DefaultContextLogger = &logger
```

Comment:
Nice, this is a lot simpler than adding middleware only to put the logger in the context